dcm2bids insertion pipeline (#752)
aces/Loris-MRI: docs/04-Scripts.md

## 4.2 - Pipeline flow

### 4.2.1 - DICOM to MINC pipeline flow

A very brief illustration of the key and expected outcomes at different steps in 
the execution of the pipeline are shown below. 

![pipeline_flow](images/pipeline_schematic_dcm2mnc.png)

This figure highlights the few 
key scripts in the pipeline execution, the order in which they call each other, 
and their interactions with the diverse libraries and utility classes. It 
therefore serves as a tool to help project developers understand the pipeline
quickly, and develop independently project customizations and debugging skills.

### 4.2.2 - DICOM to BIDS pipeline flow

A very brief illustration of the key and expected outcomes at different steps in 

> the execution of the pipeline are shown below. 

Reviewer: ```suggestion
the execution of the pipeline is shown below. 
```
(because illustration is singular)

Author: ooops, haha

![pipeline_flow_dcm2bids](images/pipeline_schematic_dcm2bids.png)

> The DICOM insertion part is identical to the DICOM to MINC pipeline flow. After the 

Reviewer: ```suggestion
The DICOM insertion steps are quite similar to the DICOM to MINC pipeline flow. After the 
```

See also: [MRI-PR#141](https://github.com/aces/Loris-MRI/pull/141) for more
  examples.

###### BIDS use case

> NIfTI volumes with their JSON side file can be **force-loaded** into LORIS by running:

Reviewer: ```suggestion
NIfTI volumes with their JSON sidecar files can be **force-loaded** into LORIS by running:
```
